.pre-commit-config.yaml

```diff
@@ -1,31 +1,6 @@
 repos:
-  - repo: https://github.com/psf/black
-    rev: 23.7.0
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.5.4
```

Comment:
**[Suggestion, non-blocking]** 0.5.5 just got released a few minutes ago 😅  https://github.com/astral-sh/ruff/releases/tag/0.5.5 But I wouldn't mind being one minor version out of date!

Reply:
Done in e1dba4b!